src/coord/src/error.rs

```diff
@@ -110,6 +117,9 @@ impl CoordError {
 impl fmt::Display for CoordError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
+            CoordError::AlterOnDisabledIndex(name) => {
+                write!(f, "cannot ALTER disabled index {}", name.quoted())
```

Comment:
The error text here is at odds with this PR which does allow ALTER on a disabled index, but only if it's to enable that index. I have no suggestions for better wording and am 100% ok keeping it as is.